orca/orca-queue-tck/src/main/kotlin/com/netflix/spinnaker/orca/q/LogMessageTask.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025 Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.q
+
+import com.netflix.spinnaker.orca.api.pipeline.Task
+import com.netflix.spinnaker.orca.api.pipeline.TaskResult
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+
+class LogMessageTask : Task {
+  private val log = LoggerFactory.getLogger(javaClass)
+
+  override fun execute(stage: StageExecution): TaskResult {
+    log.info("MDC: ${MDC.getCopyOfContextMap()}")
```

Comment:
I'maI may have missed this... but... should this REALLY be a log.info?  MDC can have a LOT of data potentially in some systems.

Reply:
This is test code.